diesel/src/macros/mod.rs

```diff
@@ -613,12 +614,6 @@ macro_rules! __diesel_table_impl {
                 Table,
```

Comment:
Any reason why this imports remain here? I think they all should be removed.

Reply:
Since I missed them, I've removed them!